django_select2/views.py

```diff
@@ -49,10 +49,10 @@ def get(self, request, *args, **kwargs):
     def get_queryset(self):
         """Get QuerySet from cached widget."""
         kwargs = {
-            model_field_name: self.request.GET.get(form_field_name)
+            model_field_name: self.request.GET.get(form_field_name) or None
```

Comment:
This change is not needed since, get will already return none if the key does not exist.
```python
>>> {}.get('does-not-exist') is None
True
```

This would only make sense, if you want to have a `None` value instead of a boolean false like empty string, `0` or `False`.

```suggestion
            model_field_name: self.request.GET.get(form_field_name)
```

Reply:
This have been used to convert `""` to `None` and prevent a `ValueError` exception on the server-side and say the server to get `id=None` object:

> `ValueError: Field 'id' expected a number but got ''.`